jekyll/_cci2/orb-author.md

```diff
@@ -9,279 +9,289 @@ version:
 - Cloud
 ---
 
+* TOC
+{:toc}
+
 ## Introduction
 
-Once you have configured your platform to work with CircleCI orbs, you are ready to begin authoring your own orb. CircleCI has attempted to make this process simple and easy, eliminating any potential confusion by enabling you to use inline templates and other pre-configured templates to quickly author a new orb. The sections below describe the process of authoring your own orb.
+  Once you have read the [Introduction to authoring an orb]({{site.baseurl}}/2.0/orb-intro) document and claimed your namespace, you are ready to develop and orb.
 
-## Orbs Quickstart
+  As orbs are simply packages of [reusable config]({{site.baseurl}}/2.0/reusing-config), orbs can be written [manually]({{site.baseurl}}/2.0/orb-author-validate-publish) as singular `yaml` files and published using our [circleci orb cli]({{site.baseurl}}/2.0/local-cli/#installation).
 
-Before you create your first orb, please see the following notes:
+  Alternatively, whether you are writing your first orb or getting ready for production level, we recommend using our Orb Development Kit to get started.
```

Comment:
@KyleTryon mind if I switch this bit round to present the orb dev kit as the suggested route?

Reply:
👍